summary33.txt

```diff
@@ -0,0 +1,100 @@
+Interface Customized wear and take off menu
+Interface Display tools required to repair wood/paper/bone/chitin items
+Content Fix Anfo and RDX charge requirements.
+I18N Load names when the language settings change.
+Infrastructure Move (de)serialization of Creature_tracker into the usual functions
+Infrastructure Remove const members from class game
+Infrastructure Remove unused catacurse functions.
+Content Add food material
+Balance Get rid of special aisle curtain item
+Content Update description of bone meal and chitin powder
+Bugfixes Fix missing headfile
+Content Add naked profession
+Content bullet pulling as quality
+Content Enable to forge a screwdriver
+Infrastructure Change nc_color to be a class
+Infrastructure Move call to init_colors into init_interface
+Infrastructure Move definition of msgtype_to_tilecolor into "cata_tiles.cpp"
+Infrastructure Move to functions from input.cpp into ncursed_def.cpp
+Infrastructure Remove the global ERR constant.
+Infrastructure Replace game::npc_by_id with critter_by_id
+Content Add reloaded 7.62x25mm to item group
+Bugfixes Add v_curtain_item to migration.json
+Content Create recipe for seed potatoes
+Content Add NPC mutation rounds.
+Bugfixes Prevent crash when repairing only frame in a vehicle.
+Bugfixes Make underbarrel flamethrower use "flammable" ammo types
+Infrastructure Load only base_colors file when it is available.
+Content Update trucks.json
+Content Update vans_busses.json
+Mods Alternate map key
+Bugfixes Correct turret weapon naming (fixes #20158).
+Bugfixes Fix NPCs giving null mission when missions run out.
+Infrastructure Switch OS X locale detection to CoreFoundation.
+Infrastructure Remove some dead mapgen code
+Infrastructure Use NULL_ID instead of magic value string
+Content Potato companion
+Content Increase firehelmet weight.
+Bugfixes Added soldering requirement
+Infrastructure Some minor doc updates
+Infrastructure Change mutation_rounds array members to proper objects.
+Infrastructure Implement common curses interface
+Infrastructure Remove include of "bionics.h" from "character.h".
+Bugfixes Fix JSON syntax highlighting
+Bugfixes Fix catacurses bugs
+Features Make all long activities abortable.
+Bugfixes Check pkg-config ncurses actually works
+Balance Remove autofire mods.
+Infrastructure Remove include of "json.h" from most headers.
+Features Draw power directly from UPS with UPS mod.
+Bugfixes Fixed null entries in mapgen
+Bugfixes Catches a few stray pixels in ChestHole's shrubs.
+Bugfixes Disallow nonempty containers with BOIL quality being used for boiling
+Bugfixes Include header in game.h and auto_pickup.h, fixing #22562
+Balance Rationalize ranged skill training.
+Infrastructure Move remaining forum links to discourse
+Bugfixes Fix bandit camp mission spawning on top of refugee center.
+Bugfixes Remove swappable storage batteries when their cases are destroyed.
+Infrastructure Update changelog up to 2474e51
+Interface Normalize colors for alternate map keys
```

Comment:
That and three below was more like `Infrastructure` as it changed color id to use same format in all jsonz.

Reply:
Fixed.